viewing_party/main.py

```diff
@@ -0,0 +1,161 @@
+def create_movie(movie_title, genre, rating):
+
+    # Create a new movie
+    new_movie = {"title": movie_title, "genre": genre, "rating": rating}
+
+    if new_movie["title"] == None or new_movie["genre"] == None or new_movie["rating"] == None:
+        return None
+    return new_movie
```

Comment:
this works great! one thing to consider: the variable new_movie is being create BEFORE the variables are validated, which means you might have spent space in memory and code time building a variable that'll ultimately be thrown out if it's not valid.

let's consider this:

```suggestion

if title and genre and rating: # if these parameters "exist"/aren't falsy
    new_movie = {"title": movie_title, "genre": genre, "rating": rating}
    return new_movie
else:
    return None
```